src/ui/public/visualize/visualize.js

```diff
@@ -54,7 +50,6 @@ uiModules
         if (!$scope.uiState) $scope.uiState = $scope.vis.getUiState();
         else $scope.vis._setUiState($scope.uiState);
 
-        $scope.vis.visualizeScope = true;
```

Comment:
Don't use `vis` object anymore to pass information to `visualization`, but use the `listen-on-change` parameter.